### PR TITLE
Doc for using bloop as the build server

### PR DESCRIPTION
**Pull request by Quafadas**
I hope that this addition to the docs, might make it easier for a beginner to get started with VSCode / mill. As far as I can tell, using bloop works reasonably well.

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 1]
```diff
@@ -912,7 +912,18 @@ object foo extends ScalaModule with ScalaMetalsSupport {
   def scalaVersion = "{example-scala-version}"
 }
 ----
---
+== Using Bloop
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:915]: I think this should belong under IDE support and next to BSP section.

```suggestion
=== Using Bloop (standalone BSP server)
```

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 2]
```diff
@@ -912,7 +912,18 @@ object foo extends ScalaModule with ScalaMetalsSupport {
...
+
+If you wish to use bloop as your build server (it is the default metals BSP implementation) with metals / vscode, you may also consider following the instructions here;
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:917]:
```suggestion
If you wish to use Bloop as your build server (it is the default metals BSP implementation) with metals / vscode, you may also consider following the instructions here: 
```

**Review by lefou — COMMENTED** [review]
Great. Documentation additions are always much appreciated! 

Please note, that in MIll 0.10, there is no need to modify the `build.sc` to use Bloop. Best would be to just delegate to the Bloop contrib plugin section (https://com-lihaoyi.github.io/mill/mill/Contrib_Plugins.html#_bloop, `xref:Contrib_Plugins.adoc#_bloop[]`) or copy its (less invasive) usage doc here.

**Comment by Quafadas** [comment]
Thankyou for the feedback. I have made the changes you suggested.

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 3]
```diff
@@ -912,7 +912,15 @@ object foo extends ScalaModule with ScalaMetalsSupport {
   def scalaVersion = "{example-scala-version}"
 }
 ----
---
+=== Using Bloop (standalone BSP server)
+
+If you wish to use Bloop as your build server (it is the default metals BSP implementation) with metals / vscode, consider using the mill bloop plugin. 
+
+https://com-lihaoyi.github.io/mill/mill/Contrib_Plugins.html#_bloop[Bloop Plugin]
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:919]: We want Antora (Asciidoctor) to handle the final link.

```suggestion
xref:Contrib_Plugins.adoc#_bloop[Bloop Plugin]
```

**Review by lefou — COMMENTED** [review]
Thank you!

You can test the generated page locally by running

```
mill docs.antora.localPages
```

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 4]
```diff
@@ -912,7 +912,15 @@ object foo extends ScalaModule with ScalaMetalsSupport {
   def scalaVersion = "{example-scala-version}"
 }
 ----
---
+=== Using Bloop (standalone BSP server)
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:915]: You need an extra empty line here

```suggestion

=== Using Bloop (standalone BSP server)
```

Comment (Quafadas) [reply docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:915]: Thankyou

Comment (lefou) [reply docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:916]: Oh, it's not the missing extra line but the line with the two dashes (`--`) which you removed before, that's missing here.

Comment (lefou) [reply docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:916]:
```suggestion
--

=== Using Bloop (standalone BSP server)
```

**Review by lefou — APPROVED** [review]
Getting documentation rendering right can be hard sometimes. 😉

**Comment by Quafadas** [comment]
Yes. I think it's clear next time will need to be done with this; 
```
mill docs.antora.localPages
```

Thankyou for your patience.

**Comment by lefou** [comment]
Thank you for improving and contributing to Mill!

**Pull request merged**